src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -218,6 +223,11 @@ class ContentSidebar extends PureComponent<Props, State> {
                 this.fetchComments(fileId);
                 this.fetchTasks(fileId);
                 this.fetchVersions(fileId);
+                if (typeof user === 'string' || typeof user === 'undefined') {
```

Comment:
Thinking about this again. If FEF can't pass us a well formed user object, then there's really no point of them passing a User ID. We can just get that with /users/me.

Reply:
Done